src/software/ai/hl/stp/tactic/passer_tactic.cpp

```diff
@@ -38,13 +38,18 @@ double PasserTactic::calculateRobotCost(const Robot& robot, const World& world)
 
 void PasserTactic::calculateNextAction(ActionCoroutine::push_type& yield)
 {
-    // Collect the ball
-    auto intercept_action = std::make_shared<InterceptBallAction>(field, ball, false);
-    do
+    // If the ball is moving, we are likely already in a live game scenario and
+    // so need to collect the ball before we can pass. If the ball is not moving,
+    // we are likely in a set play and so don't need to initially collect the ball
```

Comment:
It does work on stationary balls, but not in the "way we want" in this case.

The `InterceptBallAction` has to "run into" a stationary ball and then bring it to a stop with the dribbler. This causes 2 issues in the "set play" passing scenario:
* We move the ball before kicking. This could be seen as a double-touch, or dribbling the ball into play, both of which are not allowed in the rules
* Because we hit the ball, we trigger the callback used to trigger kicks (which only happens once per contact). This means when we try kick after making contact, it doesn't work. This bug is being fixed in #1694